stack.yaml

```diff
@@ -1,23 +1,15 @@
-resolver: lts-13.19
+resolver: lts-14.16
 
 packages:
 - .
 
 extra-deps:
-- base-noprelude-4.12.0.0
-- megaparsec-7.0.1
-- parser-combinators-1.0.0
-- cmark-0.5.6
-- text-format-0.3.2
-- fmt-0.6
 - pretty-terminal-0.1.0.0
 - roman-numerals-0.5.1.5
-- req-1.2.1
-- aeson-options-0.0.0
-- aeson-1.3.1.1
-- o-clock-1.0.0.1
+- aeson-options-0.1.0
+- o-clock-1.1.0
 
 - git: https://github.com/serokell/lootbox.git
-  commit: 34e389808e34f1cfa56f456773682325bed56d17
+  commit: 4700376b8493f6ac164461715cb72a0259148ac2 # master
```

Comment:
Probably because of this :)

Reply:
What shall we do if we update some revision in `stack.yaml`? How can we regenerate `sources.json`?

Reply:
Hmm, you gave me a good idea actually

Reply:
No, not a good idea. There's no good way to combine `niv` with `nix-prefetch-git`

Reply:
Hm, wait, actually a good idea. There is a way to combine them (of course, niv uses nix-prefetch-git)

Reply:
we could use the code from https://github.com/serokell/stack-to-nix to automatically derive the cache file from stack.yaml